util/liblog/liblog.go

```diff
@@ -11,13 +11,18 @@ type Logr interface {
 }
 
 func FromSDKLogger(l log.Logger) Logr {
-	return &lgwr{l: l}
+	return &lgwr{l}
 }
 
 type lgwr struct {
 	l log.Logger
 }
 
+// Impl implements Logr.
+func (*lgwr) Impl() any {
+	return log.NewNopLogger()
+}
```

Comment:
since we use `"cosmossdk.io/log"` we need to implement `Impl()` that returns type `any`. So instead of returning `NopLogger` if we return `nil` also it won't make any difference.

Reply:
I mean, consumers may still call `Impl()` without checking the underlaying implementation, so `nil` is not a good idea. But since it's just returning any, you'd have to make a type assertion anyway. This is okay I suppose.